core/player/src/controllers/flags/types.ts

```diff
@@ -0,0 +1,17 @@
+import { Severity } from "../../logger";
+
+/** Configuration for runtime Player behavior */
+export interface PlayerFlags {
+  /** What log level duplicate ID errors during view resolution should be raised as */
+  duplicateIDLogLevel: Severity;
+
+  /** Log level for when there are cache conflicts in view resolution (usually related to duplicate IDs) */
+  cacheConflictLogLevel: Severity;
+}
```

Comment:
Open to feedback on the flag format. For simplicity this initial pass was built to use flat flags primarily because nested flags would take more logic to merge and operate on and pulling in another dependency would be preferable to writing that logic custom but something like that would add a few kb to Player's size

Reply:
We have the `LocalDataStore` and the binding format we could use for nesting. Then do something like: 
```ts
logs.level.duplicateId
```

Reply:
I was considering using that but it seemed like overkill complexity-wise and potentially adding some overhead to get a flag if we need to go through binding parsing (even if its cached). I can write some benchmarks for it though to see what the impact would be.